pom.xml

```diff
@@ -546,13 +556,15 @@
         <axis2.version>1.6.1-wso2v37</axis2.version>
         <axiom.version>1.2.11-wso2v16</axiom.version>
 
-        <wss4j.version>1.5.11-wso2v18</wss4j.version>
+        <wss4j.version>1.5.11-wso2v18-SNAPSHOT</wss4j.version>
```

Comment:
change the snapshot to a release version

Reply:
Will do when it's merged

Reply:
Fixed